scss/fonts.scss

```
@@ -239,8 +239,8 @@
  * non-system fonts are only applied once loaded in order to
  * prevent FOIT.
  */
-$font-default-sans-fallback: 'Helvetica Neue', Helvetica, sans-serif;
-$font-default-serif-fallback: Georgia, serif;
+ $font-default-sans-fallback: -apple-system, 'Helvetica Neue', Helvetica, 'Hiragino Sans', sans-serif;
+ $font-default-serif-fallback: -apple-system, Georgia, 'Hiragino Sans', serif;
```

```suggestion
$font-default-serif-fallback: Georgia, 'Hiragino Sans', serif;
```

actually I think `-apple-system` is explicitly the sans-serif version